3.2/docker-entrypoint.sh

```diff
@@ -87,20 +87,10 @@ case "$1" in
 				[ -n "$val" ] || continue
 				echo "  $var: \"$val\"" >> config/database.yml
 			done
-		else
-			# parse the database config to get the database adapter name
-			# so we can use the right Gemfile.lock
-			adapter="$(
-				bundle exec ruby -e "
```

Comment:
This should actually work just fine without `bundle exec` (due to how the base image sets up bundles), so we should just drop that instead (keeping the `ruby -e ...` bit).